src/app/account/account-block/account-block.component.html

```diff
@@ -0,0 +1,34 @@
+<div id="header">
+    <h1>reported users</h1>
+</div>
+<div id="container-comp">
+  <div class="grid">
+    <mat-card class="card" *ngFor="let account of accountReports">
+      <div class="image-div">
+        <img [src]="getProfileImageURL(account.reported.id)"
+             (error)="handleImageError($event)"
+             class="circular-image" alt="Profile Image">
+        <mat-card-title class="time">{{ getTimeDifference(account.date) }}</mat-card-title>
```

Comment:
I think we have a pipe for this now.
```suggestion
        <mat-card-title class="time">{{ account.date | relativeTimeFilter }}</mat-card-title>
```